giskard/project.py

```diff
@@ -346,7 +347,6 @@ def _validate_label_with_target(classification_labels, target_values=None):
                 raise ValueError(f"Target column value {invalid_target_values} not declared in "
                                  f"classification_labels list: {classification_labels}")
 
-
     @staticmethod
     def _validate_classification_labels(classification_labels, model_type):
         res = None
```

Comment:
Instead of checking for __iter__ and then checking if classification_labels aren't str or dict you can just check if it's `Iterable`

```python
from typing import Iterable
isinstance(classification_labels, Iterable)
```

Reply:
Sure. I can change this if we are sure that removing this validation would not break the AI Inspect.

Reply:
I'll be more confident in checking for `Iterable` instance than for `__iter__` presence :)